energy_analyse2020.10.16_213101.R

```diff
@@ -0,0 +1,177 @@
+#set path of data and get all the subforders
+#change the path where you put the data
+setwd("D:/Rdata/2020.10.16_213101/data/nexus6p") 
+#change the path where you put the data
+projectpath="D:/Rdata/2020.10.16_213101/data/nexus6p"
+
+library(stringr)
+
+#for app with critical
+criticalfiles<-list.files(pattern = "subjects_critical")
+name_cri_chro <- strsplit(criticalfiles, "-")

```

Comment:
Why include the browser name here? If this script is simply ran with a different browser's input data it will work the same, so no two scripts and different variable names are necessary.

Reply:
When writing it, I don't find a way to extract path as string and make string into path. So I decide to put it as browser name to make it easier.

Reply:
But a static browser name is not preferable.